### ISSUE TITLE
Initial Release checklist

### ISSUE THREAD

**Issue by dayne** [acep-uaf/ak-energy-statistics-2011_2021#1]
- [x] Organizational credit in the contributors section of the README - rather than logos from each organization of the contributors instead give organizational credit / project funding credits (@dayne)
   - [x] ACEP
   - [x] ISER
   - [x] Dowl
   - [x] AEA
   - [x] UAF / UAA
   - [x] ARCTIC & ONR
   - [x] State of Alaska
- [x] Give credit to each contributor (@dayne) 
- [x] Put a LICENSE file in place and reference it in the README (CC-BY-SA)
- [x] Expand context words of this (@dayne) 
- [x] clean up the branches so the only thing we have is `main` that is locked down and acts as the primary driver for releases and the website (@ianalexmac ) 
- [x] Make the repo public so others can help (@teknofire) 
- [ ] Redirect any "new" stuff not part of the release to the `dev` branch
- [x] Create a CHANGELOG file that represents the details of "initial release" for now and will be used to acknowledge changes to future releases (fixes) to the data.
- [ ] Fully clean and validate data from 2020 and 2021. It currently has lots of missing and incorect data.
- [ ] Get the above reviewed by
   - [ ] Steve
   - [ ] Gwen
   - [ ] Sarah
- [x] Once approved by above tag a release of 2023-12-?? or as v1.0.0 (pick one) and push that release to generate the release .zip bundle
- [ ] Loop back to README and offer a full release bundle zip link on the README website (no need to -release for this change)

**Comment by eldobbins** [comment]
This release is blocked by the changes that have happened in the workbook since this issue was created.  Move its status to blocked, and make a group of new issues that should unblock it.
